### ISSUE TITLE
unresolved imports `winapi::shared::winerror`, `winapi::um::handleapi`, `winapi::um::winbase`

### ISSUE THREAD

**Issue by Dmajster** [elast0ny/mem_file#46]
when trying to compile shared_memory 0.11.1 on Windows 10 I get this error.

```
error[E0432]: unresolved imports `winapi::shared::winerror`, `winapi::um::handleapi`, `winapi::um::winbase`
  --> C:\Users\domen\.cargo\registry\src\github.com-1ecc6299db9ec823\shared_memory-0.11.1\src\windows.rs:4:9
   |
4  |         winerror::ERROR_ALREADY_EXISTS,
   |         ^^^^^^^^ could not find `winerror` in `shared`
...
8  |         handleapi::{CloseHandle, INVALID_HANDLE_VALUE},
   |         ^^^^^^^^^ could not find `handleapi` in `um`
9  |         memoryapi::{MapViewOfFile, UnmapViewOfFile, VirtualQuery, FILE_MAP_READ, FILE_MAP_WRITE},
10 |         winbase::{CreateFileMappingA, OpenFileMappingA},
   |         ^^^^^^^ could not find `winbase` in `um`
```

Other information:
windows 10 (build 17763)
rustup 1.21.1 (7832b2ebe 2019-12-20)
cargo 1.45.0-nightly (cb06cb269 2020-05-08)
rustc 1.45.0-nightly (a74d1862d 2020-05-14)

**Comment by elast0ny** [comment]
Hi, this most likely looks like some error with the winapi version... do you know what that version is ?

**Comment by Dmajster** [comment]
I'm not aware of the version as I have not ever directly worked with winapi before

**Comment by elast0ny** [comment]
Could you share a minimal example that causes the issue ? 

I cant reproduce this locally and it wasn't caught by the automatic builds either...

p.s. : I updated the Cargo.toml winapi dependency to add all the features I'm using in `shared_memory`.  Please try to build from master and let me know if that fixed the issue.
```toml
[dependencies]
shared_memory = { git = "https://github.com/elast0ny/shared_memory-rs" }
```

**Comment by Dmajster** [comment]
sure, but it is literally an empty project with shared_memory as it's only dependency and the mutex example code.

https://github.com/Dmajster/shared_memory_issue

[edit] adding the dependency then threw errors about raw_sync missing and worked normally when I added raw_sync to cargo.toml. I tried then doing the same at my old project and it didn't work. It appears to be some other issue on my end. 

thank you!
